crates/next-dev/tests/integration.rs

```diff
@@ -164,7 +165,13 @@ async fn create_browser(
             .args(vec!["--auto-open-devtools-for-tabs"]);
     }
 
-    let (browser, mut handler) = Browser::launch(config_builder.build()?).await?;
+    let config = config_builder.build()?;
+    let (browser, mut handler) = match Browser::launch(config.clone()).await {
+        Ok(r) => Ok(r),
+        // retry once if failed to launch
+        Err(_) => Browser::launch(config).await,
+    }
+    .context("Launching browser failed")?;
```

Comment:
Can we reuse the logic in https://github.com/vercel/turbo/pull/2692/files instead? Maybe moving these retry_async utils in a shared module for next-dev testing.

Reply:
done